docs/user/filter_expressions.rst

```diff
@@ -0,0 +1,205 @@
+Filter Expressions
+==================
+Filter expressions describe which users a :ref:`recipe <recipes>` should be
+executed for. They're executed locally in the client's browser and, if they
+pass, the corresponding recipe is executed. Filter expressions have access to
+information about the user, such as their location, locale, and Firefox version.
+
+Filter expressions are written using a language called JEXL_. JEXL is an
+open-source expression language that is given a context (in this case,
+information about the user's browser) and evaluates a statement using that
+context. JEXL stands for "JavaScript Expression Language" and uses JavaScript
+syntax for several (but not all) of its features.
+
+.. note:: The rest of this document includes examples of JEXL syntax that has
+   comments inline with the expressions. JEXL does **not** have any support for
+   comments in statements, but we're using them to make understanding our
+   examples easier.
+
+.. _JEXL: https://github.com/TechnologyAdvice/Jexl
+
+JEXL Basics
+-----------
+The `JEXL Readme`_ describes the syntax of the language in detail; the following
+section covers the basics of writing valid JEXL expressions.
+
+.. note:: Normally, JEXL doesn't allow newlines or other whitespace besides
+   spaces in expressions, but filter expressions in Normandy allow arbitrary
+   whitespace.
```

Comment:
IMO not until they do a release and/or we're using the version that supports this.